### DISCUSSION TITLE
Missing recurrence options in latest version (2.0.2)

### DISCUSSION THREAD

**Discussion by ganesshkumar** [iSoron/uhabits#1068] [Feature Requests] [answered]
I was using the latest version from Play Store and today I downloaded 2.0.2 apk from fdroid and installed it on my phone. 

Once I updated to 2.0.2, I noticed that I am missing some recurrence option in the latest version. I used to set `350/365 days` as a recurrence option (I was trying to avoid a bad habit here, allowing myself only to do it 15 times per year). In the new recurrence picker, there is no way we can do this.  

![Screenshot_20210817-225904](https://user-images.githubusercontent.com/2135089/129773831-96b19371-2487-4aa7-aa1f-d3a624846b8e.jpg)

One simple way is to add another options call `__ times per year` in the new dialog. Can we have this?

**Comment by hiqua** [comment, marked as answer]
Duplicate: https://github.com/iSoron/uhabits/discussions/961
